apps/shell/src/app/components/demo/table-demo.component.ts

```
@@ -0,0 +1,299 @@
+        },
+      },
+      {
+        iconClass: 'fa_solid:trash',
+        classCss: 'delete-action',
+        method: (row: DemoData) => this.handleDelete(row),
+      },
+    ];
+  }
+
+  handleEdit(row: DataSourceMaterialTable<any>) {
+    row.editable != row.editable
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)